### PR TITLE
[skip appveyor] add symmetrica

### PR DESCRIPTION
**Pull request by isuruf**

### CODE REVIEW

File: recipes/symmetrica/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+test:
+  commands:
+    - test -f ${PREFIX}/include/symmetrica/macro.h
+    - test -f ${PREFIX}/lib/libsymmetrica.a
+
+about:
+  home: http://www.algorithm.uni-bayreuth.de/en/research/SYMMETRICA/index.html
+  license: ISC
+  license_file: {{ RECIPE_DIR }}/LICENSE
+  summary: The C-library  cddlib is a C implementation of the Double Description Method of Motzkin et al.
```
Comment (dopplershift) [inline recipes/symmetrica/meta.yaml]: These two lines need updating I suspect.

Comment (isuruf) [inline recipes/symmetrica/meta.yaml]: Done in bfe90af

**Comment by isuruf** [comment]
@dopplershift, @ocefpaf ready for review now

File: recipes/symmetrica/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: http://www.algorithm.uni-bayreuth.de/en/research/SYMMETRICA/SYM2_0_tar.gz
+  sha256: bf52788dedc14c482e89f5e7efe8c60864a633314ddd446dd4602d5fdaca0ee2
+
+  # patches are not added here, because the files in the tarball have permission 400 and
```
Comment (ocefpaf) [inline recipes/symmetrica/meta.yaml]: Can't you fix the permission and commit that? (I am fine doing it in the feedstock.)

Comment (isuruf) [inline recipes/symmetrica/meta.yaml]: What do you mean by that?

Comment (ocefpaf) [inline recipes/symmetrica/meta.yaml]: Oh. It is the tarball and not the patches. I read it wrong. Not sure what we can do then...

**Pull request merged**